README.md

```diff
@@ -337,7 +343,7 @@ $('#mergely').mergely('unmarkup');
 
 ### update
 
-Manual update; recalculates diff and applies new settings.
+Manual update, recalculates the diff and applies new settings.
```

Comment:
The `;` to `,` is not appropriate — it introduces a run-on sentence.

Maybe this could be reworked into, e.g.

> Manually updates the editor by recalculating the diff and applying new settings

WDYT?